### PR TITLE
Update tests and dependencies

### PR DESCRIPTION
**Pull request by hkdobrev**

### CODE REVIEW

File: tests/src/ApiTest.php [hunk 1]
```diff
@@ -205,15 +205,10 @@ public function testGetSendUri()
...
-            'Openbuildings\Postmark\Api',
-            array(
-                'getSendUri'
-            ),
-            array(
-                'POSTMARK_API_TEST'
-            )
-        );
+        $apiMock = $this->getMockBuilder('Openbuildings\Postmark\Api')
+            ->setMethods(array('getSendUri'))
```
Comment (EZarkov) [inline tests/src/ApiTest.php]: Why you use array() instead of []?

Comment (hkdobrev) [inline tests/src/ApiTest.php]: @EZarkov This library still supports PHP 5.3. There's no good reason to drop support unless for shorter array syntax. Even if we wanted to drop 5.3 support, that would be a breaking change and would require a version bump.

Comment (EZarkov) [inline tests/src/ApiTest.php]: LGTM

**Comment by EZarkov** [comment]
LGTM

**Pull request merged**